docs/config.json

```diff
@@ -736,6 +736,11 @@
               "title": "Azure PostgreSQL & MySQL",
               "slug": "/database-access/guides/azure-postgres-mysql/"
             },
+            {
+              "title": "Azure SQL Server (Preview)",
+              "slug": "/database-access/guides/azure-sql-server-ad/",
+              "forScopes": ["oss", "enterprise"]
```

Comment:
why not "cloud"? I wasn't aware of the "forScopes" option before I saw this

Reply:
I misunderstood the Cloud architecture, but this will be supported on it too. I will update the docs with the Cloud sections. Thanks for pointing it out.